src/App.js

```diff
@@ -0,0 +1,20 @@
+import { Routes, Route } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Countries from './components/Countries/Countries';
+import Header from './components/Header/Header';
+import CountryData from './components/CountryData/CountryData';
+import './App.css';
+
+function App() {
```

Comment:
- [ ] Deployed app is displaying correctly, however the repository version compiles successfully but is displaying blank screen. Kindly check and update the version of your code on the repository accordingly, and the next code reviewer will pick it from there.